docs/conf.py

```diff
@@ -15,6 +15,7 @@
 import os
 import sys
 
+sys.path.append(os.path.abspath('./sphinx-ext/'))
```

Comment:
what... is this for?

Reply:
::scrolls back up from lower:: ohhhh I seeeeeeeeeeeeeeee toctreefilter!

Reply:
I added a custom Sphinx extension that allows us to filter the table-of-context.  This tells Sphinx where to look for the extension.  The extension was copy-paste-edited from another source and I gave appropriate credit in the header.

Reply:
The UI did not refresh and I did not see your comment :)